utils/measurements/distance.py

```diff
@@ -52,6 +52,14 @@ def m(self) -> Decimal:
     def km(self) -> Decimal:
         return convert_unit(self, C.M_TO_KM)
 
+    @property
+    def cm(self) -> Decimal:
+        return convert_unit(self, C.M_TO_CM)
```

Comment:
I just had a thought; perhaps the return types of all of these should be `DistanceType`

```suggestion
    @property
    def cm(self) -> 'DistanceType:
        return self.__class__(convert_unit(self, C.M_TO_CM))
```